Felienne/hedy
# Improve UI to let user know they are in single-adventure mode

**Description**
In this PR we add a text to the adventure tabs view to inform the user when they are in single adventure mode.

**Fixes**
This PR fixes #2146

**How to test**
Navigate to single adventure mode. For example by navigating to `/adventure/turtle`. Verify that the string given in the adventure tabs is correct and only appears when in this mode. When leaving the mode the string should disappear.

![Schermafbeelding 2022-03-14 om 15 30 59](https://user-images.githubusercontent.com/38155424/158194019-97566c1f-853c-43c8-ae2b-549662a0f2a3.png)
